lib/Newsletter/Editor/PostContentManager.php

```diff
@@ -68,7 +68,7 @@ public function filterContent($content, $displayType, $withPostClass = true) {
 
     $content = strip_tags($content, implode('', $tagsNotBeingStripped));
     if ($withPostClass) {
-      $content = str_replace('<p', '<p class="' . self::WP_POST_CLASS . '"', WPFunctions::get()->wpautop($content));
+      $content = preg_replace('/<p( class=\".+\")?/', '<p class="' . self::WP_POST_CLASS . '"', WPFunctions::get()->wpautop($content));
```

Comment:
I assume that we need to replace the class attribute here and make sure there are no 2 class attributes, correct?
I think this will not work if the class attribute will be placed other than first e.g. `<p id="anchor" class="custom-class">`.  So maybe we should remove the original class and then add the new one. What do you think?

It might be handy to have a comment here. I don't even know why we need to replace the class. So maybe if you learned why it is needed could you please add a comment so that the future us know?

I also noticed that there is [a test for the class](https://github.com/mailpoet/mailpoet/blob/b39dac75d6a729f5cff25a38e3d9b104f0711a34/tests/integration/Newsletter/Editor/PostContentManagerTest.php) so it would be nice to have this covered in the test.